sentry/src/main/java/io/sentry/Trace.java

```diff
@@ -0,0 +1,104 @@
+package io.sentry;
+
+import com.jakewharton.nopen.annotation.Open;
+import io.sentry.protocol.SentryId;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+@Open
+public class Trace {
+  public static final String TYPE = "trace";
+
+  /** Determines which trace the Span belongs to. */
+  private final @NotNull SentryId traceId;
+
+  /** Span id. */
+  private final @NotNull SpanId spanId;
+
+  /** Id of a parent span. */
+  private final @Nullable SpanId parentSpanId;
+
+  /** Short code identifying the type of operation the span is measuring. */
+  protected @Nullable String op;
+
+  /**
+   * Longer description of the span's operation, which uniquely identifies the span but is
+   * consistent across instances of the span.
+   */
+  protected @Nullable String description;
+
+  /** Describes the status of the Transaction. */
+  protected @Nullable SpanStatus status;
+
+  /** A map or list of tags for this event. Each tag must be less than 200 characters. */
+  protected @Nullable Map<String, String> tags;
+
+  public Trace() {
+    this(new SentryId(), new SpanId(), null);
+  }
+
+  public Trace(
+      final @NotNull SentryId traceId,
+      final @NotNull SpanId spanId,
+      final @Nullable SpanId parentSpanId) {
+    this.traceId = traceId;
+    this.spanId = spanId;
+    this.parentSpanId = parentSpanId;
+  }
+
+  public @NotNull String toTraceparent() {
```

Comment:
```suggestion
  public @NotNull String toSentryTrace() {
```

Dont' we have sampling decision?

Reply:
Sampling has not been implemented yet.